tests/test_selected_word_counter.py

```diff
@@ -81,6 +82,11 @@ def test_extract_files_run():
     finally:
         shutil.rmtree(test_dir_converted)
 
+        for item in os.listdir(test_dir):
```

Comment:
Why is this necessary? Is this not deleting the test_files?

Reply:
only the test folder of which the original files were extracted

Reply:
tested this multiple times so i am sure that it works